Source/WebKit/UIProcess/Launcher/glib/ProcessLauncherGLib.cpp

```diff
@@ -144,7 +144,7 @@ void ProcessLauncher::launchProcess()
     if (!setCloseOnExec(socketPair.client))
         RELEASE_ASSERT_NOT_REACHED();
 
-    g_child_watch_add(pid, [](GPid pid, gint, gpointer) { g_spawn_close_pid(pid); }, nullptr);
+    g_child_watch_add(pid, [](GPid pid, gint status, gpointer) { GUniqueOutPtr<GError> error; if(!g_spawn_check_exit_status (status, &error.outPtr())) g_warning("%s", error->message); g_spawn_close_pid(pid); }, nullptr);
```

Comment:
Please read the webkit coding style guidelines: https://webkit.org/code-style-guidelines/
You haven't addressed the formatting, it's best to not have all that on one line. You can format the lambda over multiple lines, or factor it out in an auto variable and pass that. I'd do the latter,

```
auto watchFunc = [](GPid ...) { 
      .. with line breaks ..;
 };
g_child_watch_add(pid, watchFunc, nullptr);
```